### PR TITLE
Feature/create single tweet component

### PR DESCRIPTION
**Pull request by ArtemLigerko**

### CODE REVIEW

File: src/components/SingleTweet.tsx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+  fill: rgb(83, 100, 113);
+  padding: 0;
+  transition-duration: 0.2s;
+  border-radius: 50%;
+`;
+const IconChat = styled(Icons)`
+  &:hover {
+    fill: rgb(29, 155, 240);
+  }
+`;
```
Comment (Mykyta-Pysarev) [inline src/components/SingleTweet.tsx]: You could use one styled component with props for Icon, IconBg etc.

**Review by Mykyta-Pysarev — APPROVED** [review]
Overall seems ok.

**Comment by ArtemLigerko** [comment]
image

File: package.json [hunk 2]
```diff
@@ -21,8 +21,10 @@
     "bootstrap": "^5.2.0",
     "react": "^18.2.0",
     "react-bootstrap": "^2.5.0",
+    "react-bootstrap-icons": "^1.8.4",
```
Comment (KirillSerg) [inline package.json:28]: it will be on my merg

File: package.json [hunk 3]
```diff
@@ -21,8 +21,10 @@
...
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.34.0",
+    "react-icons": "^4.4.0",
```
Comment (KirillSerg) [inline package.json:31]: are they really needed?

**Review by KirillSerg — APPROVED** [review]

File: src/components/SingleTweet.tsx [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+  margin-right: 12px;
+  border-radius: 50%;
+`;
+const NameSection = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+`;
+const NameWrapper = styled.div``;
+const Name = styled.div`
```
Comment (Tretmax) [inline src/components/SingleTweet.tsx]: maybe UserName

File: package.json (hunk 3 above)
Comment (alexandrtovmach) [reply package.json:31]: also interested

File: src/components/SingleTweet.tsx [hunk 5]
```diff
@@ -0,0 +1,224 @@
+import { Dropdown } from "react-bootstrap";
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  max-width: 598px;
+  padding: 0 16px;
```
Comment (alexandrtovmach) [inline src/components/SingleTweet.tsx]: margins and paddings in bootstrap can be managed in next way:
https://getbootstrap.com/docs/4.0/utilities/spacing/

please fix here and everywhere

File: src/components/SingleTweet.tsx [hunk 6]
```diff
@@ -0,0 +1,224 @@
+import { Dropdown } from "react-bootstrap";
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  max-width: 598px;
```
Comment (alexandrtovmach) [inline src/components/SingleTweet.tsx]: what's a magic number?

File: src/components/SingleTweet.tsx (hunk 4 above)
Comment (alexandrtovmach) [reply src/components/SingleTweet.tsx]: Username, without upper N

File: src/components/SingleTweet.tsx [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+  border-radius: 50%;
+`;
+const NameSection = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+`;
+const NameWrapper = styled.div``;
+const Name = styled.div`
+  font-weight: bold;
```
Comment (alexandrtovmach) [inline src/components/SingleTweet.tsx]: font customization can be solved with bootstrap Text modifications
https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics

fix here and everywhere

File: src/components/SingleTweet.tsx [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+            </NameWrapper>
+            <Dropdown>
+              <SDropdownToggle variant="success" id="dropdown-basic">
+                <svg viewBox="0 0 24 24">
+                  <g stroke="grey">
+                    <circle cx="5" cy="10" r="2" />
+                    <circle cx="12" cy="10" r="2" />
+                    <circle cx="19" cy="10" r="2" />
+                  </g>
+                </svg>
```
Comment (alexandrtovmach) [inline src/components/SingleTweet.tsx]: I thought we agreed to use icons instead of SVG?

File: src/components/bootstrap.min.css [hunk 9]
```diff
@@ -0,0 +1,10463 @@
+@charset "UTF-8"; /*!
```
Comment (alexandrtovmach) [inline src/components/bootstrap.min.css]: what a hell is that?

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]
remove bootstrap.min.css from components folder

File: src/components/SingleTweet.tsx (hunk 6 above)
Comment (ArtemLigerko) [reply src/components/SingleTweet.tsx]: like in Tweeter

File: src/components/bootstrap.min.css (hunk 9 above)
Comment (ArtemLigerko) [reply src/components/bootstrap.min.css]: fogot delete it

File: public/index.html [hunk 10]
```diff
@@ -25,6 +25,8 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <!-- CSS only -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
```
Comment (alexandrtovmach) [inline public/index.html]: delete that

File: public/index.html [hunk 11]
```diff
@@ -25,6 +25,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <!-- CSS only -->
```
Comment (alexandrtovmach) [inline public/index.html:28]:
```suggestion
```

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**